week-2/debug/2.js

```diff
@@ -12,3 +12,4 @@ console.log(`The last digit of 806 is ${getLastDigit(806)}`);
 
 // This program should tell the user the last digit of each number.
 // Explain why getLastDigit is not working properly - correct the problem
+//The getLastDigit is not working well because the variable is interfering with the argument of the function call. For it to work the variable will be placed as an argument in the function.
```

Comment:
So you've understood the problem and updated the code too. Delete the code once you've corrected it